framework/docker/hyperlane/agent.go

```diff
@@ -1,11 +1,31 @@
 package hyperlane
 
-import "github.com/celestiaorg/tastora/framework/docker/container"
+import (
+	"context"
 
+	"github.com/celestiaorg/tastora/framework/docker/container"
+)
+
+// AgentType defines the type of Hyperlane agent
+type AgentType string
+
+const (
+	AgentTypeRelayer   AgentType = "relayer"
+	AgentTypeValidator AgentType = "validator"
```

Comment:
would be cool to see the validator agent used for a multisig tx at some point in the future, but not sure how priority there is for that rn